src/Adapter/LocalStorage.php

```diff
@@ -197,6 +197,14 @@ public function getURL($path)
 
         return $this->webUrl.'/'.$path;
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    public function getExpiringURL($path, $expiresAt) {
+        // Local storage cannot provide expiring URLs. Falling back...
```

Comment:
Imagine you're using this bundle so that your production code talks to an S3 bucket, but locally you can fall back to using the local file system.

Making the LocalStorage adapter throw an exception forces the developer to write "if s3 do x, else do y/try do x, catch do y" logic that the bundle should be doing for them, IMHO.